### PR TITLE
Feature/cy3 net import

### PR DESCRIPTION
**Pull request by chrtannus**
It's importing the Cy3 json format for now, until we have a **cx-to-json** converter that supports **CX v1** (or a cx1-to-cx2 converter).
Features so far:
- Lists the current networks from current Cy3 session:
  - If no networks: show _info_ message
  - If Cytoscape 3 not running: show _error_ message
- After the selected network is imported:
  - just removes current _elements_ and _data_
  - adds new _elements_ and _data_
  - apply the **grid layout** if the imported network did not have a view; or **fit to screen**, otherwise (uses imported node positions).
  - the imported **style** format (Cy3 json) does not work, so I'm just converting/testing 'node color' for now.
- The header **title** is updated with the imported network's name

### CODE REVIEW

File: src/client/components/network-editor/controller.js [hunk 1]
```diff
@@ -32,6 +32,47 @@ export class NetworkEditorController {
...
   }
 
+  /**
+   * Replaces the current network with the passed one.
+   * @param {Object} [elements] Cytoscape elements object
+   * @param {Object} [data] Cytoscape data object
+   * @param {Object} [style] Optional Cytoscape Style object
+   */
+  setNetwork(elements, data, style) {
+    this.cy.remove(this.cy.$().remove());
```
Comment (maxkfranz) [inline src/client/components/network-editor/controller.js:42]:
```suggestion
    this.cy.elements().remove();
```

You have an extra remove call here.  You can just do one on the elements.

File: src/client/components/network-import/cy3-network-import-dialog.js [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+
+  handleCancel() {
+    this.onClose();
+  }
+
+  handleOk() {
+    const netId = this.state.value;
+    this.onClose();
+
+    fetch(`${CY3_URL}/v1/networks/${netId}/views`)
```
Comment (maxkfranz) [inline src/client/components/network-import/cy3-network-import-dialog.js:47]: You could make this more manageable with async/await.

You  just need to mark `async handleOk()`, and then you can use `let response = await fetch(...)` etc. in a more linear, sequential style that looks a lot like synchronous code.

See also https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Asynchronous/Async_await

**Review by maxkfranz — COMMENTED** [review]
Looks great.  I've added some minor comments that you may find useful in future.

File: src/client/components/network-import/cy3-network-import-dialog.js (hunk 2 above)
Comment (chrtannus) [reply src/client/components/network-import/cy3-network-import-dialog.js]: Cool! I've just pushed some changes. Let me know what you think. Thanks!

**Pull request merged**